components/vf-card/vf-card.config.yml

```diff
@@ -1,5 +1,5 @@
-# The title shown on the component page
-title: Card
+# The card_title shown on the component page
+card_title: Card
```

Comment:
I think the `title` replacement on line 1, 2 here are find-replace typos.

Reply:
how embarrassing

Reply:
fixed in latest commit